shared/cfg/base.cfg

```diff
@@ -88,6 +88,20 @@ netdst = virbr0
 # present in VM, default is 'no'
 flexible_nic_index = no
 
+# Add device intel-iommu
+# intel_iommu = yes
+# Set intel-iommu options
+# iommu_intremap = on/off/auto  (on by default)
+# iommu_device-iotlb = on/off  (on by default)
```

Comment:
Keep consistent, no matter what it is in the option, all param use `_` ?   `iommu_device_iotlb` ?
Or, use the exact option name, both are ok but keep consistent ~